README.md

```diff
@@ -19,11 +19,11 @@ When writing Python code for service mapping or other NSO development Pylint is
 Since usually you build packages as part of a CI job with that the Pylint will also check the Python code. With that you prevent that syntactic errors or code ends up in a merge request since for a successful pipeline run you have to fix them.
 ## Use Pylint for NSO Python package analysis
 
-You will learn how to use Pylint on an existing NSO package that uses Python to implement mapping logic. In the `src/` folder you will find the sample loopback package you will work with in this lab.
+You will learn how to use Pylint on an existing NSO package that uses Python to implement mapping logic. In the `~/src/` folder you will find the sample loopback package you will work with in this lab.
 
 ### Modify package Makefile to include pylint
 
-First add a recipe for running Pylint to the package Makefile. Open Makefile under `src/loopback/src/Makefile` and add the the call for `pylint` target to the `all:` target on the first line:  
+First add a recipe for running Pylint to the package Makefile. Open Makefile under `~/src/loopback/src/Makefile` and add the the call for `pylint` target to the `all:` target on the first line:  
```

Comment:
Consider that once the repo is added to the playground, it will be automatically cloned. The playground does a git clone under the src directory. This means, the code cloned is under a directory with the name of the repository. 

For example with the current repo, the path under the playground will be:
```bash
~/src/ll-test/loopback/src/Makefile
```

Once this code is pushed to the final repository update the paths.

Reply:
I think it will be better to keep the structure you will get with the playground. Avoid moving the directory like we are doing provisionally with `mv ll-test/* src/`